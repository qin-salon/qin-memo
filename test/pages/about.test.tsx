import { render } from "@testing-library/react";
import About from "src/pages/about";

describe("About page", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<About />, {});
    expect(asFragment()).toMatchSnapshot();
  });
});
