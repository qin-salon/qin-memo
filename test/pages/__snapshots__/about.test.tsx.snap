// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page matches snapshot 1`] = `
<DocumentFragment>
  <header>
    <div
      class="bg-white dark:bg-gray-800"
    >
      <h1
        class="text-gray-900 dark:text-white"
      >
        Title
      </h1>
      <button
        class="text-white dark:text-gray-900 bg-gray-800 dark:bg-white"
      >
        Change Theme
      </button>
      <nav>
        <a
          class="text-gray-800 dark:text-gray-100"
          href="/"
          style="display: inline-block; padding: 12px;"
        >
          Home
        </a>
        <a
          class="text-gray-800 dark:text-gray-100"
          href="/about"
          style="display: inline-block; padding: 12px;"
        >
          About
        </a>
      </nav>
    </div>
  </header>
  <main
    class="bg-red-100 dark:bg-gray-700"
  >
    <h2
      class="text-gray-800 dark:text-white"
    >
      About
    </h2>
    <button
      class="sample"
    >
      clsxサンプル
    </button>
    <button
      class="sample bold"
    >
      clsxサンプル(propsに応じてスタイル変更)
    </button>
  </main>
  <footer
    class="bg-white dark:bg-gray-800"
  >
    <small
      class="text-gray-800 dark:text-white"
      lang="en"
    >
      @ 2021 memo.qin.sh
    </small>
  </footer>
</DocumentFragment>
`;
